app/js/components/manageApps/AddonList.jsx

```diff
@@ -39,15 +39,14 @@ export const AddonList = ({handleDownload, install, appList, openPage, openModal
                 id="delete-icon-wrapper"
               >
                 {install == false ? 
-                  <i className="glyphicon glyphicon-trash text-danger delete-icon"  onClick={openModal(app)}/> :
-                  <i className="glyphicon glyphicon-download-alt text-primary install-icon"  onClick={(e) => handleDownload(e)}/> 
+                  <i className="glyphicon glyphicon-trash text-danger delete-icon" id="icon-btn" onClick={openModal(app)}/> :
+                  <i className="glyphicon glyphicon-download-alt text-primary install-icon" id="icon-btn" onClick={(e) => handleDownload(e)}/> 
```

Comment:
Can we reduce break this line into new lines as it exceeds the 180 limit per line.

Reply:
This was an oversight, thanks for pointing it out.